control/autoware_mpc_lateral_controller/src/mpc_lateral_controller.cpp

```diff
@@ -304,9 +306,13 @@ trajectory_follower::LateralOutput MpcLateralController::run(
   publishPredictedTraj(predicted_traj);
   publishDebugValues(debug_values);
 
-  const auto createLateralOutput = [this](const auto & cmd, const bool is_mpc_solved) {
+  const auto createLateralOutput =
+    [this](
+      const auto & cmd, const bool is_mpc_solved,
+      const auto & cmd_horion) -> trajectory_follower::LateralOutput {
```

Comment:
Fix typos.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8977/commits/3aea463c6b04a4bd911af38e16e0000ebe66861d